core/string/ustring.cpp

```diff
@@ -1026,6 +1019,22 @@ String String::camelcase_to_underscore(bool lowercase) const {
 	return lowercase ? new_string.to_lower() : new_string;
 }
 
+String String::to_camel_case() const {
+	String s = this->to_pascal_case();
+	if (!s.is_empty()) {
+		s[0] = s.to_lower()[0];
+	}
+	return s;
+}
+
+String String::to_pascal_case() const {
+	return this->capitalize().replace(" ", "");
+}
+
+String String::to_snake_case() const {
+	return this->camelcase_to_underscore().replace(" ", "_").strip_edges();
+}
```

Comment:
Doesn't `camelcase_to_underscore()` already change `" "` to `_`? If not, is there any reason for it not to, and is there still any use case for it instead of using `to_snake_case()`?

Reply:
Because now `camelcase_to_underscore` is used only in `capitalize` and `to_camel_case` methods, I think it makes sense to make it private. I did not replace spaces with underscores in `_camelcase_to_underscore`, because `capitalize` replaces underscores with spaces back.